pkg/sdk/poc/generator/templates/show_object_method.tmpl

```diff
@@ -0,0 +1,6 @@
+{{- /*gotype: github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/poc/generator.ResourceHelperMethod*/ -}}
```

Comment:
let's have dedicated templates for both functions and let's move more logic there (e.g. for ObjectType method we should require only structName, for identifier we should require id type and args and handle it inside the template)

Reply:
done